h2o-algos/src/main/java/hex/deeplearning/DeepLearningModel.java

```diff
@@ -72,6 +73,14 @@ public DeepLearningModelOutput(DeepLearning b) {
     }
   } // DeepLearningModelOutput
 
+    @Override
+    public void computeEffectiveParameters() {
```

Comment:
should this also call super.?

Reply:
it could, but I think its not necessary. by calling super.computeEffectiveParameters() firstly, it would set distribution, stopping_metric, fold_assignment and categorical encoding to null, and then after super. set them to effective values by respective logic. there is assert(_dist._family != DistributionFamily.AUTO); in both DeepLearningModel constructors, so it wont let effective distribution = AUTO, and in other lines it also cannot happen that it will let some of those effective parameters = AUTO.